packages/bpk-component-card/src/BpkCardWrapper.tsx

```diff
@@ -31,27 +37,127 @@ type Props = {
   className?: string | null;
   backgroundColor: string;
   header: ReactNode;
+  body?: {
+    text: string;
+    openBtnLabel: string;
+    closeBtnLabel: string;
+    link?: string;
+    linkText?: string;
+    moreInfoBtnColor?: string;
+  };
 };
 
 const BpkCardWrapper = ({
   backgroundColor,
+  body,
   card,
   className = null,
   header,
 }: Props) => {
   const classNames = getClassName('bpk-card-wrapper', className);
 
-  return (
-    <CardContext.Provider value={{ elevated: false }}>
+  const [isBodyOpen, setIsBodyOpen] = useState(false);
```

Comment:
For this I would call this

```suggestion
  const [isExpanded, setIsExpanded] = useState(false);
```